patches/chrome-android-java-src-org-chromium-chrome-browser-ntp-IncognitoNewTabPage.java.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPage.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPage.java
+index 5273ff0bf0ced1788d003b4823283dc4358d5ced..1a73b31cff37f9a54bd8f4403c9ecb1169227496 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPage.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPage.java
+@@ -33,7 +33,7 @@ public class IncognitoNewTabPage
+     private Activity mActivity;
+ 
+     private String mTitle;
+-    protected IncognitoNewTabPageView mIncognitoNewTabPageView;
```

Comment:
actually are these changes even necessary? `BraveIncognitoNewTabPageView` is a subclass of `IncognitoNewTabPageView` and `inflate` will create the correct class based on https://github.com/brave/brave-core/pull/3509/files#diff-ea88c09faf6ae0fc5fb51f009c05d03eR10

Reply:
Yeah, you're right these aren't necessary at all - good catch!